time TimeStamp
int32 batteryLevel
float32 batteryVoltage
int32 currentSpeed
int32 desiredSpeed
int32 currentSpeedLeft
int32 desiredSpeedLeft
int32 currentSpeedRight
int32 desiredSpeedRight
int32 currentAngle
int32 desiredAngle
float32 yaw
float32 pitch
float32 roll
bool sysCalib 
bool gyrCalib 
bool accCalib 
bool magCalib 
float32 accelX
float32 accelY
float32 accelZ
int32 turningLights
int32 illumination
int32 illumination_f
int32 illumination_r
int32 lights
int32 error1
int32 error2
int32 sensor
string version
bool teleop
bool charging
bool picOK
float32 angular_rate_sp
float32 angular_rate_curr
uint64 UDPTimeStamp
uint64 QRTimeStamp
uint32 LastControlMsg
bool AutoPilot
bool Inferring
string Imei
string Iccid
float64 Latitude
string LatHemisphere
float64 Longitude
string LonHemisphere
float64 Altitude
float64 Accuracy
string UTCTime
string UTCDate
int32 Satellites
int32 Sim
int32 rssi
bool validGPS
string provider
string accessTechnology
string band
bool ModuleConnected
int32 speaker_volume
